.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,25 +3,35 @@
 > Describe what changes this pull request makes to the repository
 
 **Check list**
+
 - [ ] contains breaking changes
 - [ ] adds new feature
 - [ ] modifies existing feature (bug fix or improvements)
 - [ ] relies on other tasks
 - [ ] documentation changes
 - [ ] tested on mobile devices
 
+**Notes for release notes:**
```

Comment:
Maybe to simplify this sentence to "Release notes"

Reply:
@bobo-k2 updated in [this](https://github.com/AstarNetwork/astar-apps/pull/630/commits/036997db1181c07d37b20705470fb6521ba5981c) commit.